trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/standardfunctions/StandardFunctionHandler.java

```diff
@@ -886,6 +892,65 @@ private ExpressionResult getNondetStringOrNull(final ILocation loc) {
 		return builder.build();
 	}
 
+	private Result handleVaStart(final IDispatcher main, final IASTFunctionCallExpression node, final ILocation loc,
+			final String name) {
+		final IASTInitializerClause[] arguments = node.getArguments();
+		checkArguments(loc, 2, name, arguments);
+		final ExpressionResultBuilder builder = new ExpressionResultBuilder();
+		final ExpressionResult arg0 =
+				mExprResultTransformer.transformDispatchDecaySwitchRexBoolToInt(main, loc, arguments[0]);
+		builder.addAllExceptLrValue(arg0);
+		// TODO: We do not actually use this parameter, this could be unsound.
+		final ExpressionResult arg1 =
+				mExprResultTransformer.transformDispatchDecaySwitchRexBoolToInt(main, loc, arguments[1]);
+		builder.addAllExceptLrValue(arg1);
+		final Expression dst = arg0.getLrValue().getValue();
+		if (!(dst instanceof IdentifierExpression)) {
+			throw new UnsupportedSyntaxException(loc, "The first argument of " + name + " has to be an identifier.");

```

Comment:
Probably ok for now. If we encounter cases where this fails, we could perhaps change the code, so that `dst` is not assigned the pointer, but instead points to a memory location where we write the pointer.

Reply:
Yes, this could be done. I did however not encounter such an example yet and I am not sure if anyone actually does that 🙃 (it would be really strange code imo)